python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputAttachmentsSchemaRenderer.tsx

```diff
@@ -26,7 +25,6 @@ function EditableAttachmentRenderer({
   onRemoveAttachment?: () => void;
 }) {
   const [showUploader, setShowUploader] = useState(attachment?.data == null);
-  const {readOnly} = useContext(AIConfigContext);
```

Comment:
Did this get disconnected or is it not needed? i.e. if rendering an input with attachments, user is not able to modify attachments right now, right?

Reply:
Its disconnected, its not used anywhere in this file. I'll take a quick look but maybe its used in the subcomponents

Reply:
looks like this got erroneously added and was never used: https://github.com/lastmile-ai/aiconfig/commit/f3c043cf06a63ff469ad57e04b4aa71f2e1c35a3#diff-313b20da4217207c9e335e51d5fb50043b71d909c204cd13470b15526d4e2578R29

Reply:
Ah, right. Looks like AttachmentContainer handles readOnly case. I think it's possible AttachmentUploader still needs to be updated to handle readOnly but that's a separate thing